composer.json

```diff
@@ -17,13 +17,13 @@
     ],
     "require": {
         "php": "^7.1",
-        "opis/closure": "^3.0",
+        "opis/closure": "^3.4.2",
```

Comment:
First version to work at all on PHP 7.4 due to `stream_set_option` implementation is 3.4.2. 

NB 3.5.0 added support for short closures, but we don't need this.